src/main/java/at/hannibal2/skyhanni/features/nether/reputationhelper/dailyquest/DailyQuestHelper.kt

```diff
@@ -198,7 +201,9 @@ class DailyQuestHelper(val reputationHelper: CrimsonIsleReputationHelper) {
         display.addAsSingletonList("")
         display.addAsSingletonList("§7Daily Quests (§e$done§8/§e5 collected§7)")
         if (done != 5) {
-            quests.mapTo(display) { renderQuest(it) }
+            val filteredQuests: List<Quest> = if (config.hideComplete) quests.filter { it.state != QuestState.COLLECTED }
```

Comment:
`val filteredQuests = quests.filter { !config.hideComplete || it.state != QuestState.COLLECTED }`